src/node/markdown/markdown.ts

```diff
@@ -51,13 +60,14 @@ export interface MarkdownOptions extends MarkdownIt.Options {
   headers?: HeadersPluginOptions | boolean
   sfc?: SfcPluginOptions
   theme?: ThemeOptions
-  languages?: ILanguageRegistration[]
+  languages?: LanguageInput[]
   toc?: TocPluginOptions
   externalLinks?: Record<string, string>
   cache?: boolean
   component?: ComponentPluginOptions
   math?: boolean | any
   container?: ContainerOptions
+  shikijiTransformers?: ShikijiTransformer[]
```

Comment:
should we scope shikiji related stuff to something like `markdown.shikiji.[theme, languages, transformers, alias, ...]` 👀  -- deprecate markdown.theme, remove markdown.languages (as the new syntax is anyway different and people using that will need to update)

Reply:
I was thinking that, if you don't mind another breakage.

`theme` and `langs` should be compactible most of the cases tho.

Reply:
The current version looks fine too I guess. If you're planning to add many new features to shikiji in near future, then we could scope them. For just 3-4 options, flatter config is fine.